src/utils/Utils.ts

```diff
@@ -65,6 +66,19 @@ export default class Utils {
     });
   }
 
+  public static getEponymousAttributeValue(resourceInstance: ResourceInstance): string {
+    if (!resourceInstance.resourceType.eponymousAttribute) {
+      return `${resourceInstance.id}`;
+    }
+    const eponymousAttribute = resourceInstance.resourceType.attributes.filter( (resourceTypeAttribute) => {
```

Comment:
Consider using [`find`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/find) instead of [`filter`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) :)

Reply:
you are right, when looking for a singe result, find is more expressive. I changed it

Reply:
👍